src/chttpd_cors.erl

```diff
@@ -306,6 +316,35 @@ get_cors_config(#httpd{cors_config = undefined}) ->
 get_cors_config(#httpd{cors_config = Config}) ->
     Config.
 
+cors_config(Host, Key, Default) ->
+    config:get(cors_section(Host), Key,
+                     config:get("cors", Key, Default)).
+
+cors_section(Host0) ->
+    {Host, _Port} = split_host_port(Host0),
```

Comment:
@iilyak that's some groovy stuff... ok, let's keep it here as it was there, to not to increase complexity, and leave the cleaning for a next round.